src/main/java/opendota/Parse.java

```diff
@@ -306,7 +307,18 @@ public void onChatEvent(Context ctx, CDOTAUserMsg_ChatEvent message) {
         entry.value = value;
         output(entry);
     }
-    
+
+    // New chat event
+    @OnMessage(CDOTAUserMsg_ChatMessage.class)
+    public void onAllChatMessage(Context ctx, CDOTAUserMsg_ChatMessage message) {
+        int type = message.getChannelType();
+        Entry entry = new Entry(time);
+        entry.slot = message.getSourcePlayerId();
+        entry.type = (type == 11) ? "chat" : String.valueOf(type);
```

Comment:
did you see any non-11 events get emitted in a test replay?

Reply:
nope - my guess is the other types (teamchat/coach chat) arestripped from replays